core/Pos/Core/Block/Main/Chain.hs

```diff
@@ -33,6 +33,7 @@ import           Pos.Core.Txp (TxPayload, TxProof, mkTxProof)
 import           Pos.Core.Update (UpdatePayload, UpdateProof, mkUpdateProof)
 import           Pos.Crypto (Hash, PublicKey, hash)
 
+
```

Comment:
Please avoid whitespace changes if they are the only changes in a file and are not really important (this one is certainly not important).